### PR TITLE
Search perf

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts [hunk 1]
```diff
@@ -1,21 +1,28 @@
 ﻿$(() => {
   const searchBox = $("#searchBox");
 
+  // clear the filter so we don't have to deal with the case of a user navigating backwards and
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts:4]: Why don't we just run filter function again when the user navigates backwards?

Comment (JoshLove-msft) [reply src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts:4]: We can but then the page jumps after it is initially rendered.

Comment (pakrym) [reply src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts:4]: meh, it's okiosh

File: src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts [hunk 2]
```diff
@@ -1,28 +1,44 @@
...
-        context.mark(searchText, {
-          done: function () {
-            context.not(":has(mark)").closest("tr").hide();
-          }
-        });
-      }
-    },
-      // use a timeout to prevent bad perf for holding down backspace or really fast typing
-      200);
+    setTimeout(filter, 300);
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts:15]: isn't 300 too much?

**Review by pakrym — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts (hunk 2 above)
Comment (JoshLove-msft) [reply src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts:11]: From my testing, this seems to be the sweet spot.

**Pull request merged**